tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.nuspec

```
@@ -28,8 +28,8 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="build\Datadog.Trace.Trimming.props" target="build\Datadog.Trace.Trimming.props" />
-    <file src="build\Datadog.Trace.Trimming.xml" target="build\Datadog.Trace.Trimming.xml" />
+    <file src="build\Datadog.Trace.Trimming.props" target="buildTransitive\Datadog.Trace.Trimming.props" />
```

Wait, why do we use a `nuspec` file for this one? 😅

---

Don't ask, I blame @tonyredondo 😂 
```suggestion
    <file src="build\Datadog.Trace.Trimming.props" target="build\Datadog.Trace.Trimming.props" />
    <file src="build\Datadog.Trace.Trimming.props" target="buildTransitive\Datadog.Trace.Trimming.props" />
```